pyproject.toml

```
@@ -1,2 +1,85 @@
 [tool.black]
 line-length = 120
+
+[tool.pylint."MESSAGES CONTROL"]
+disable = [
+    "arguments-differ",
+    "broad-except",
+    # "broad-exception-raised",          # Only available in later pylint
+    "c-extension-no-member",             # seems to be weird false pos
+    "consider-iterating-dictionary",     # lol no
+    "consider-using-f-string",           # don't care
+    "duplicate-code",                    # dont' care
+    "eval-used",                         # needed
+    "exec-used",                         # needed
+    "fixme",
+    "global-statement",                  # needed
+    "import-outside-toplevel",           # needed
+    "inconsistent-return-statements",    # don't care
+    "invalid-name",                      # SUPER don't care
+    "keyword-arg-before-vararg",         # literally nonsensical
+    "line-too-long",                     # black takes care of my line lengths
+    "missing-class-docstring",
+    "missing-function-docstring",
+    "missing-module-docstring",
+    "missing-timeout",                   # TODO: fix
+    "no-else-break",                     # i prefer this actually
+    "no-else-continue",                  # ditto
+    "no-else-raise",                     # ditto
+    "no-else-return",                    # ditto
+    "pointless-string-statement",        # fine as alt comment syntax
+    "redefined-builtin",
+    "superfluous-parens",                # don't care
+    "too-few-public-methods",            # dumb
+    "too-many-arguments",                # dumb
+    "too-many-boolean-expressions",      # needed
+    "too-many-branches",                 # needed
+    "too-many-instance-attributes",      # dumb
+    "too-many-lines",
+    "too-many-locals",                   # dumb
+    "too-many-nested-blocks",
+    "too-many-return-statements",
+    "too-many-statements",
+    "unsubscriptable-object",            # false positives
+    "unsupported-binary-operation",      # false pos on type sigs, plus mypy catches anyway
+    "use-dict-literal",                  # don't care
+    "useless-return",                    # need it for mypy
+]
+
+[tool.ruff]
+target-version = "py37"
+ignore = [
+    "E501",
+    # bare accept is fine
+    "E722",
+    # don't care
+    "E741",
+    # don't care
+    "E743",
+    # handling unused imports elsewhere
+    "F401",
+    # ditto
+    "F405",
+    # flake8 doesn't understand typing overloads
+    "F811",
+    # caught by pylint and I'm not silencing two things
+    "F821",
+    "N802",
+    "N803",
+    "N806",
+    "N812",
+    "N815",
+    "N816",
+    "N999",
+]
+line-length = 120
+select = [
+    "E",
```

Add the functionality of `rbialon/flake8-annotations`...
```suggestion
    "ANN",
    "E",
```

Also, `"PL"` for pylint